hackathon_site/dashboard/frontend/src/components/team/TeamCardAdmin/TeamCardAdmin.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styles from "./TeamCardAdmin.module.scss";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Card from "@material-ui/core/Card";
+
+interface CardProps {
+    teamCode: string;
+    members: string[];
+}
+
+export const TeamCardAdmin = ({ teamCode, members }: CardProps) => {
+    const title = `Team ${teamCode}`;
```

Comment:
You can put this directly in the jsx because there's not point in creating a constant for just a small string